pycqed/measurement/waveform_control/pulse_library.py

```diff
@@ -209,7 +209,7 @@ def chan_wf(self, chan, tvals):
         t_rel = tvals - tvals[0]
         wave *= np.cos(
             2 * np.pi * (self.frequency * t_rel + self.phase / 360.))
-        return wave
+        return wave - wave[0]
```

Comment:
I understand that the waveform should start with 0, but is it correct to shift the whole waveform? Shouldn't it rather just set wave[0] to 0?

Reply:
For me both seem like legitimate solutions. 

I have to admit, however, that I did not see problematic behavior if the waveforms are non-zero in the beginning. I am not sure about the status, whether we still believe that non-zero waveforms might lead to malfunctioning of the 5014. May @antsr knows more here.

If this is the case, then we should fix all waveforms such that they do start with zero.

Reply:
It is an issue with the 5014, so waveforms need to start with 0. This should not have to be done on a pulse level though, but at the element level in the segment class

Reply:
Then I will revert this change!